tasks/quality_control/advanced_metrics/task_busco.wdl

```diff
@@ -33,19 +34,25 @@ task busco {
     if [ -f ~{samplename}/short_summary.specific.*.~{samplename}.txt ]; then
 
       # grab the database version and format it according to BUSCO recommendations
-      cat ~{samplename}/short_summary.specific.*.~{samplename}.txt | grep "dataset is:" | cut -d' ' -f 6,9 | sed 's/,//' | sed 's/ / (/' | sed 's/$/)/' | tee DATABASE
+      # pull line out of final specific summary file
+      # cut out the database name and date it was created
+      # sed is to remove extra comma and to add parentheses around the date and remove all tabs
+      # finally write to a file called DATABASE
```

Comment:
And this for giving a breakdown of that one liner--always helpful to have this kind of documentation